orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -6147,3 +6147,33 @@ def forward(self, x):
         torch.onnx.export.reset_mock()
 
         del os.environ["ORTMODULE_CACHE_DIR"]
+
+
+def test_reciprocal_gradient():
+    class ReciprocalModel(torch.nn.Module):
+        def __init__(self):
+            super().__init__()
+
+        def forward(self, x):
+            return 1 / x
+
+    def run_step(model, x):
+        prediction = model(x)
+        loss = prediction.sum()
+        loss.backward()
+        return prediction, loss
+
+    device = "cuda"
+    pt_model = ReciprocalModel().to(device)
+    ort_model = ORTModule(
+        copy.deepcopy(pt_model), DebugOptions(log_level=LogLevel.WARNING, save_onnx=True, onnx_prefix="reciprocal")
```

Comment:
```suggestion
        copy.deepcopy(pt_model)
```

Reply:
Thanks. I missed removing this after debugging.